pkg/controller/util.go

```diff
@@ -65,12 +65,12 @@ func (c *Controller) podFromKey(key interface{}) (*v1.Pod, error) {
 func (c *Controller) checkPVC(pvc *v1.PersistentVolumeClaim, get bool) (bool, error) {
 	// check if we have proper AnnEndPoint annotation
 	if !metav1.HasAnnotation(pvc.ObjectMeta, AnnEndpoint) {
-		glog.Infof("checkPVC: annotation %q not found, skipping pvc\n", AnnEndpoint)
+		glog.V(Vdebug).Infof("checkPVC: annotation %q not found, skipping pvc\n", AnnEndpoint)
 		return false, nil
 	}
 	//check if the pvc is being processed
 	if metav1.HasAnnotation(pvc.ObjectMeta, AnnImportPod) {
-		glog.Infof("checkPVC: pvc annotation %q exists indicating it is being or has been processed, skipping pvc\n", AnnImportPod)
+		glog.V(Vadmin).Infof("checkPVC: pvc annotation %q exists indicating it is being or has been processed, skipping pvc\n", AnnImportPod)
```

Comment:
`Vadmin` level logs probably shouldn't mention the function name

Reply:
ya, probably right. Will change all Vadmin and Vuser to not mention func name.